CHANGELOG.md

```diff
@@ -14,6 +14,11 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [2.6.4] - 2020-11-10
```

Comment:
I think we should bump it to minor version as it brings back `target` and `source` fields. Also, the change, even though non breaking, still has some meat in it.
```suggestion
## [2.7.0] - 2020-11-10
```